tools/ansible/roles/dockerfile/templates/supervisor_rsyslog.conf.j2

```diff
@@ -59,6 +59,15 @@ stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 
+[eventlistener:rsyslog-4xx-recovery]
+command=/awx_devel/tools/scripts/rsyslog-4xx-recovery
+buffer_size = 100
+events=PROCESS_LOG_STDERR
+priority=0
+autorestart=true
+stdout_events_enabled = true
```

Comment:
I thought this flag needed to be on the service you want to monitor. I expected it under `[program:awx-rsyslogd]`

Reply:
it feels like 
```
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
```
and 
```
stdout_events_enabled = true
stderr_events_enabled = true
```
is equivalent 
and rsyslogd section already have 
```
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
```

Reply:
```An event type emitted when a process writes to stdout or stderr. The event will only be emitted if the file descriptor is not in capture mode and if stdout_events_enabled or stderr_events_enabled config options are set to true. ```

nvm